src/pages/Finances.tsx

```diff
@@ -23,8 +46,12 @@ const Finances = () => {
         </MainGrid>
       </MainGridWrapper>
       <SecondGridWrapper>
-        <Title as="h2">Announcements</Title>
-        <AnnouncementContainer className="col-span-2" type="financial" />
+      {events.length > 0 && (
+          <>
+          <Title as="h2">Announcements</Title>
+          <AnnouncementContainer className="col-span-2" type={announcementType} events={events} />
```

Comment:
it doesn't look like you need or use announcementType in the container component.. remove it if it isn't being used.